sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusHostBuilderExtensions.cs

```diff
@@ -81,6 +81,10 @@ public static IWebJobsBuilder AddServiceBus(this IWebJobsBuilder builder, Action
                         "SessionHandlerOptions:MaxConcurrentSessions",
                         options.MaxConcurrentSessions);
 
+                    options.MaxMessageBatchSize = section.GetValue(
+                        "BatchOptions:MaxMessageCount",
+                        options.MaxConcurrentSessions);
```

Comment:
This doesn't look right..

Reply:
We missed the property in the change:
https://github.com/Azure/azure-sdk-for-net/pull/18330/files#diff-abd5707be3a68d80538eadf936318f75e383409245f763b6c87678622b5aa1a8

Also added to back comparability tests.

Reply:
The issue is we are falling back to MaxConcurrentSessions instead of MaxBatchSize.

Reply:
Indeed , I haven't noticed that default value was copy/pasted from the line above. Updated the PR.